gnomad-google-project/README.md

```diff
@@ -18,6 +18,14 @@ This module is designed to easily spin up a google cloud project in a configurat
   - compute.admin
   - dataproc.worker
   - storage.objectAdmin
+  - serviceusage.serviceUsageConsumer
+- For the provided Primary User, grant the following IAM roles:
+  - artifactregistry.admin
+  - editor
+  - iap.tunnelResourceAccessor
+- If a hail batch service account is provided, grant the following IAM roles:
+  - storage.objectAdmin
+  - artifactregistry.read
```

Comment:
what about `storage.legacyBucketWriter` (for the tmp buckets) and `storage.objectViewer`/`storage.legacyBucketReader` for other project-related buckets? (based on permissions that we granted Lily's Hail Batch SA)